README.md

```diff
@@ -43,7 +45,7 @@ To build binary: ```mvn package```
 - which will produce  ```tool/target/datacommons-import-tool-1.0-SNAPSHOT-jar-with-dependencies.jar```
```

Comment:
my bad ... could you s/1.0-SNAPSHOT/0.1-alpha.1/ ... (https://github.com/datacommonsorg/import/pull/16)

Reply:
done